spring-integration-jdbc/src/main/resources/org/springframework/integration/jdbc/store/channel/schema-h2.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE INT_CHANNEL_MESSAGE (
+	MESSAGE_ID CHAR(36) NOT NULL,
+	GROUP_KEY CHAR(36) NOT NULL,
+	CREATED_DATE BIGINT NOT NULL,
+	MESSAGE_PRIORITY INT,
+	MESSAGE_SEQUENCE BIGINT NOT NULL,
+	MESSAGE_BYTES LONGVARBINARY,
+	REGION VARCHAR(100) NOT NULL,
+	constraint INT_CHANNEL_MESSAGE_PK primary key (GROUP_KEY, MESSAGE_ID, REGION)
+);
+
+CREATE INDEX INT_CHANNEL_MSG_DATE_IDX ON INT_CHANNEL_MESSAGE (CREATED_DATE, MESSAGE_SEQUENCE);
+CREATE INDEX INT_CHANNEL_MSG_PRIORITY_IDX ON INT_CHANNEL_MESSAGE (MESSAGE_PRIORITY DESC, CREATED_DATE, MESSAGE_SEQUENCE);
+
+CREATE SEQUENCE INT_MESSAGE_SEQ START WITH 1 INCREMENT BY 1;
```

Comment:
Your IDE is still have vulnerabilities with "New line in the end" :smile: 
Will fix on merge.